configs/edison/config.20171102.beta3rc02_1850.ne30_oECv3_ICG.edison

```diff
@@ -138,9 +139,5 @@ baseDirectory =  /global/project/projectdirs/acme/ACMEv0_lowres/B1850C5_ne30_v0.
 polarPlot = False
 
 [streamfunctionMOC]
-## options related to plotting the streamfunction of the meridional overturning
```

Comment:
I'm removing this because the changes I've made elsewhere mean we shouldn't need this chunking anyway.